### ISSUE TITLE
[Content group cards]: React – VO differs from other card instances

### ISSUE THREAD

**Issue by oliviaflory** [carbon-design-system/carbon-for-ibm-dotcom#8090]
### Description

Voice over is reading Content group cards differently than other card components like Card group or Content block cards. Is this the expected behavior or should all cards be read the same way?

When in Content group cards, VO is announcing the beginning and end of a single card as a `group`. 

**Content group cards order:**
- Lorem ipsum group
- Heading level 3
- Lorem ipusum 
- Visited link, lorem ipsum
- end of, Lorem ipsum, group

Where as Card group or Content block cards does not announce a single card as a group and goes straight into the Heading.

**Card group / Content block card order:**
- Heading level 3
- Lorem ipusum 
- Visited link, lorem ipsum

**Video recording** (sorry you can hear me and not the VO)

https://user-images.githubusercontent.com/45692622/150403957-5aacfde7-b6be-4b02-92ac-02e728110809.mov

_This is present in Production_

### Component(s) impacted

Any component that uses content group cards 
– Content group cards
– Content block mixed

### Browser

Chrome

### Carbon for IBM.com version

1.30.0 rc 0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.COM

### Package

@carbon/ibmdotcom-react

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/react/?path=/story/components-content-group-cards--default

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
